packages/js/src/ai-assessment-fixes/components/ai-assessment-fixes-button.js

```diff
@@ -85,26 +106,28 @@ const AIAssessmentFixesButton = ( { id, isPremium } ) => {
 
 	// Add tooltip classes on mouse enter and remove them on mouse leave.
 	const handleMouseEnter = useCallback( () => {
-		// Add tooltip classes on mouse enter
-		setButtonClass( "yoast-tooltip yoast-tooltip-w" );
-	}, [] );
+		const direction = isEnabled ? "yoast-tooltip-w" : "yoast-tooltip-nw";
```

Comment:
Maybe it would be better to do this based on the length of the label 😅 But I did not want to introduce a constant here, so rather did it like this.